x-pack/test/fleet_api_integration/config.ts

```diff
@@ -15,7 +15,8 @@ import { defineDockerServersConfig } from '@kbn/test';
 // example: https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fpackage-storage/detail/snapshot/74/pipeline/257#step-302-log-1.
 // It should be updated any time there is a new Docker image published for the Snapshot Distribution of the Package Registry.
 export const dockerImage =
-  'docker.elastic.co/package-registry/distribution:b6a53ac9300333a4a45f3f7d350c9aed72061a66';
+  process.env.FLEET_PACKAGE_REGISTRY_DOCKER_IMAGE ||
```

Comment:
could you document this in our README? https://github.com/elastic/kibana/blob/master/x-pack/plugins/fleet/README.md#api-integration-tests

Reply:
Added in https://github.com/elastic/kibana/pull/101911/commits/527fdef8fda495c4ec9237b62ff0624a4d50e9c1